openscm/timeseries_converter.py

```diff
@@ -293,11 +314,33 @@ def _convert(
         if len(values) < 3:
             raise InsufficientDataError
 
+        try:
+            return self._convert_unsafe(values, source_time_points, target_time_points)
+        except ValueError:
+            error_msg = (
+                "Target time points are outside the source time points, use an "
+                "extrapolation type other than None"
+            )
+            raise InsufficientDataError(error_msg)
+
+    def _convert_unsafe(
+        self,
+        values: np.ndarray,
+        source_time_points: np.ndarray,
+        target_time_points: np.ndarray,
+    ) -> np.ndarray:
+        """Same as ``self._convert`` but without error checking
```

Comment:
Proper documentatin missing here (Just copy from `_convert`, which should probably “wrap _convert_safe while checking for...”)